changelog/march2024/2024-03-22-kubernetes-changed-routed-ips-now-default-on-new-kubern.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Routed IPs now default on new Kubernetes clusters
+status: changed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-03-22
+category: containers
+product: kubernetes
+---
+
+[Routed IPs](https://www.scaleway.com/en/news/routed-ips-are-coming-to-all-scaleway-products/) are coming to all Scaleway products.
```

Comment:
```suggestion
[Routed IPs](https://www.scaleway.com/en/news/routed-ips-are-coming-to-all-scaleway-products/) will soon be available for all Scaleway resources!
```

Reply:
Again I think it's more than just available, they are becoming default and they are "coming" in the sense that soon you won't be able to choose to have NAT IPs any more

Reply:
I've updated my comment!